### ISSUE TITLE
Reviewed By Bug

### ISSUE THREAD

**Issue by jjlynch2** [spawaskar-cora/cora-docs#497]
I think there may be a bug with the Reviewed By column when searching for elements.

I've been inventorying elements for Ploesti, and I noticed when searching by the provenance 1, some of the elements are listed as having been reviewed by myself. However, I have not reviewed the element. I have however updated the designator information after creating the element. Is this why Reviewed By lists my name?

What exactly is the Reviewed By column supposed to indicate? Would this indicate a peer review or anyone who updates an element?

**Comment by clegarde** [comment]
Yes, there is some issue with "Inventoried by" and "Reviewed by" columns. "Inventoried by" is populated before the "Inventory" box is checked - maybe fills based on who created it? Then the "Inventoried at" column fills when the "Inventory" box is checked.
![Screen Shot 2019-08-14 at 8 36 22 PM](https://user-images.githubusercontent.com/7350474/63067180-52521a00-bed3-11e9-840c-4c08eaeec8aa.png)
"Reviewed by" is similar to this where the column is the person that checked the "Inventoried" box and the "Reviewed at" column is blank until the "Reviewed" box is checked - which will then update with the "Reviewed by" column with whoever checked the "Reviewed" box.

**Comment by mei215** [comment]
![1](https://user-images.githubusercontent.com/33871387/63354561-45498680-c32a-11e9-9534-25fc7f62c075.JPG)
There is also an issue with the "Updated By" field. When I accept peer reviewer changes--meaning I updated an element--the "Reviewed At" and "Updated At" times change, but the "Updated By" field stays the same--the Peer Reviewer's name, not the person who last made an update.

**Comment by Katie-East** [comment]
This is related to ticket #467 

I would also like to add that if the inventoried box is checked after the review is complete for an element, even if no changes have been made, this deletes all reviewer information. I can understand why this is designed this way, but it unforgiving for errors or if other projects have different bussiness rules about when an invetory is officiall complete (before or after peer review). 

![cora reviewed by issues](https://user-images.githubusercontent.com/46726506/63437314-ea7a6280-c3ef-11e9-965b-dde2801a8000.PNG)
